### DISCUSSION TITLE
对交易的思考 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#155] [Announcements]
# 对交易的思考 · BMPI

投资交易已十年，分析我对交易的理解。

https://www.bmpi.dev/self/thinking-in-trade/

**Comment by maggch97** [comment]
只看K线的低频交易，没有基本面分析，基本就是靠天吃饭。历史回撤好看只是运气，就像你的几个A股组合，2023年之后都不行了。

没有做空，即使策略有一些alpha收益，也全都被市场的beta给亏完了。
